plugins/API/API.php

```diff
@@ -624,8 +626,6 @@ public function getSuggestedValuesForSegment($segmentName, $idSite)
                     );
                     if (!empty($segment) && preg_match('/^' . implode('|',$remove) . '/', $segment)) {
                         $values[] = urldecode(urldecode(str_replace($remove, '', $segment)));
-                    } else {
-                        $values[] = $row->getColumn('label');
```

Comment:
removing that actually requires a segment metadata to be set, right? should we maybe mention that in the doc block of `setSuggestedValuesApi`

Reply:
done 👍